editoast/src/modelsv2/prelude/update.rs

```diff
@@ -0,0 +1,343 @@
+use diesel::result::Error::NotFound;
+
+use crate::error::EditoastError;
+use crate::error::Result;
+use crate::models::Identifiable;
+
+use crate::modelsv2::prelude::*;
```

Comment:
Since we already are in the `prelude` module (a sub-module of it), I would rather avoid making a `*` import here. Maybe import exactly all the items you need instead?